src/containers/shared/components/TransactionTable/styles.scss

```diff
@@ -61,11 +61,10 @@
     }
 
     @include for-size(desktop-up) {
-      width: 130px;
```

Comment:
this moved wasnt needed

Reply:
weird, must have been an issue with a merge, reverting this change